dclab/definitions/feat_const.py

```diff
@@ -118,6 +118,13 @@
     # Fun fact: If we had decided to compute it from the convex contour,
     # then we would have close to none pixelation effects ¯\_(ツ)_/¯.
     ["volume", "Volume [µm³]"],
+    ["area_um_raw", "Area [µm²] from a raw contour"],
+    ["per_um_raw", "Perimeter [µm] from a raw contour"],
+    ["deform_raw", "Deformation from a raw contour"],
+    ["eccentr_prnc", "Eccentricity from a raw contour"],
+    ["per_ratio", "Ratio of raw and cvx perimeters (inversed convexity)"],
+    ["s_x", "Simmetry ratio left-right"],
```

Comment:
- I think `s_x` is too unspecific. Could we please change this to `sym_x` and `sym_y`?
- There is a type "Simmetry" vs "Symmetry"

Reply:
s_x is how it was called by Maik in the paper. that is why i used that naming.

Reply:
Ok, please correct the typo.

Reply:
Please also add a comment above the definition of `s_x` that this is how it was defined by Herbig et al.
Is it this one? https://doi.org/10.1063/1.5027197